examples/communication/grandparent_to_grandchild/README.md

```diff
@@ -0,0 +1,11 @@
+# Grandparent-to-Grandchild Example
+
+A simple example of updating a grandchild component from a grandparent using a shared state through the ContextProvider.
```

Comment:
I think this example doesn't show much of value that hasn't been seen in `grandchild_to_grandparent`. Maybe rename the above to `gradchild_with_grandparent` since actually in that one the communication is two-sided.

This example basically uses the context API again to read state from its grandparent, which by itself has been demonstrated before. If you want, include a message in the grandchildren there that also displays the overall count?

Reply:
Done